runtime/major_gc.c

```diff
@@ -1380,7 +1333,12 @@ static void cycle_major_heap_from_stw_single(
                  "swept_words %"ARCH_INTNAT_PRINTF_FORMAT"d",
                  heap_words, not_garbage_words, swept_words);
 
-    if (caml_stat_space_overhead.heap_words_last_cycle != 0) {
+    static struct {
+      intnat heap_words;
+      intnat not_garbage_words;
+    } last_cycle = {0, 0};
```

Comment:
Ooh, shiny, a static structure in the middle of a function! (Are we confident that the C compilers we use support this well?)

Reply:
This syntax predates C99. If a compiler does not support this, it's a severe bug in the compiler and "someone else's problem"™.

Reply:
If I declared it at the start of the function it would be C89 :-)